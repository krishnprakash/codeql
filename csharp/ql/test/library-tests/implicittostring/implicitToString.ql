import csharp

from MethodCall c
where c.isImplicit()
select c
