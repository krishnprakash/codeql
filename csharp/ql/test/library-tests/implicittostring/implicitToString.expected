| implicitToString.cs:31:27:31:35 | call to method ToString |
| implicitToString.cs:33:22:33:30 | call to method ToString |
| implicitToString.cs:35:22:35:30 | call to method ToString |
| implicitToString.cs:40:23:40:42 | call to method ToString |
